records_mover/db/driver.py

```diff
@@ -170,14 +195,10 @@ def set_grant_permissions_for_users(self, schema_name: str, table: str,
                 perms_sql = f'GRANT {perm_type} ON TABLE {schema_and_table} TO {user_name}'
                 db.execute(perms_sql)
 
+    # TODO: which interface should this be on?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker